restservice/src/main/java/org/itevents/service/sendmail/SendGridMailService.java

```diff
@@ -0,0 +1,49 @@
+package org.itevents.service.sendmail;
+import com.sendgrid.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Service;
+
+import javax.inject.Inject;
+
+/**
+ * Created by max on 29.07.15.
+ */
+@Service("mailService")
+public class SendGridMailService implements MailService {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+    private static final String FROM_MAIL = "events@juja.com.ua";
+    private static final String SUBJECT_MAIL = "IT Events";
+
+    @Inject
+    private SendGrid sendgrid;
+
+    public void sendMail(String htmlLetter, String emailAddress){
+
+        SendGrid.Email email = createMail(htmlLetter, emailAddress);
+        boolean status = send(email);
+
+        if (!status) {
+            LOGGER.error("Email sending error: ", emailAddress);
+        }
+
+    }
+
+    public boolean send(SendGrid.Email email) {
+        try {
+            return sendgrid.send(email).getStatus();
+        } catch (SendGridException e) {
+            return false;
```

Comment:
What will be if public send will be called separately and fails? In current revision log will be written only if send will be called by sendMail. Maybe public is excess?